modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/TxStateTableStorage.java

```diff
@@ -48,11 +47,15 @@ public interface TxStateTableStorage extends AutoCloseable {
      * Destroy transaction state storage.
      *
      * @param partitionId Partition id.
-     * @return Future.
      * @throws StorageException In case when the operation has failed.
      */
-    CompletableFuture<Void> destroyTxStateStorage(int partitionId) throws StorageException;
+    void destroyTxStateStorage(int partitionId) throws StorageException;
```

Comment:
I don't think that this is what I meant. You didn't compare it with partitions code, right?